certs/key.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Just seems overall that it would be better to just leave the readme explaining how to get here rather than distributing a default cert even if the intention is for testing.